src/Host/Client/Impl/Extensions/RStringExtensions.cs

```diff
@@ -35,7 +46,33 @@ public static class RStringExtensions {
             for (int i = 1; i < s.Length - 1; ++i) {
                 char c = s[i];
                 if (escape) {
-                    sb.Append(c);
+                    // https://stat.ethz.ch/R-manual/R-devel/library/base/html/Quotes.html
+                    switch (c) {
```

Comment:
that would be producing the equivalent of `"\\t"` instead of the `"\t"` we need.